libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -24,6 +24,16 @@ void AP_Mount_Backend::set_roi_target(const struct Location &target_loc)
     _frontend.set_mode(_instance, MAV_MOUNT_MODE_GPS_POINT);
 }
 
+// set_sys_target - sets system that mount should attempt to point towards
+void AP_Mount_Backend::set_sysid_target(uint8_t sysid)
+{
+    // set the target gps location
+    _state._sysid = sysid;
```

Comment:
sorry to be annoying but could we rename  the state's "sysid" to "target_sysid"?  I was initially confused thinking that this was the mount's system id rather than the system-id of the thing the mount should point to.

Also if possible could we rename "set_sysid_target" to "set_target_sysid"?

Reply:
Done.